db_stress_tool/db_stress_test_base.cc

```diff
@@ -2497,10 +2505,13 @@ Status StressTest::TestCheckpoint(ThreadState* thread,
         fault_fs_guard->EnableThreadLocalErrorInjection(
             FaultInjectionIOType::kMetadataRead);
       }
-      assert(my_s.ok());
-
-      for (const auto& f : files) {
-        fprintf(stderr, " %s\n", f.c_str());
+      if (!my_s.ok()) {
+        fprintf(stderr, "Fail to GetChildren under %s due to %s\n",
+                checkpoint_dir.c_str(), my_s.ToString().c_str());
+      } else {
+        for (const auto& f : files) {
```

Comment:
is it possible for `GetChildren()` partially fail? (e.g. fill some in `files` and return non-ok)

Wondering if you want to print files even when the my_s is not ok().

Reply:
It might be misleading though to print some partial result so I decided to not to.